### PR TITLE
Move version to libfaad/version.h file

### PR DESCRIPTION
**Pull request by eustas**
That makes it easier for using from different build systems.

`bootstrap` script updates `configure.ac` with version from `version.h`.

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
Could you also remove the `utils/win32/ac2ver.*` stuff and its usage while at it, please?

File: VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+2.10.1
```
Comment (fabiangreffrath) [inline VERSION:1]: No EOL at EOF.

Comment (eustas) [reply VERSION:1]: It is should fine both ways. Added.

**Comment by eustas** [comment]
Changed the concept. Now `libfaad/version.h` is the source of truth for version. Build systems will pick it up from there.
It is still possible to override package version via `CFLAGS`.

**Comment by fabiangreffrath** [comment]
Fine, thanks! So, for releases we will have to edit this one header file and call `./bootstrap` once to get it merged into `configure.ac`, right?

**Comment by eustas** [comment]
Correct.
Alternatively we could have version in `configure.ac` always set to e.g. `0.0.0`, and do not forget to reset it after running `bootstrap`. Currently this is will not be convenient, because `autotools` is the only build system... Later we could revise that.

**Pull request merged**